src/configs/settings.ts

```diff
@@ -307,7 +317,7 @@ export interface PlanAction {
 
 /** Plan Goal detailQuantity */
 export interface PlanGoaldetailQuantity {
-  comparator: '>=';
+  comparator: '>=' | '<=';
```

Comment:
@ciremusyoka why are we adding `'<='`?

Reply:
@moshthepitt this is based on [this file](https://docs.google.com/spreadsheets/d/1kGPgyF0eJdag19FFL1Oog2gdphdPtKobY4EQ0gMgtgw/edit#gid=1805975071) line 262.

Reply:
Cool, thanks